package.json

```diff
@@ -114,6 +114,9 @@
     "tempy": "0.2.1",
     "webpack": "3.12.0"
   },
+  "resolutions": {
+    "fsevents": "1.2.9"
+  },
```

Comment:
It is works only for yarn, no sure it is good idea

Reply:
Didn't know that 🤔 
But there is a `yarn.lock`
This implies for me that yarn should be used

Reply:
When someone installs Prettier from GitHub (e.g. to get the latest not yet released version), they can be using NPM: `npm install https://github.com/prettier/prettier.git --save`

Reply:
By the way, it only seems to work on mac ^^'
So it's not acceptable